### PR TITLE
[10.0][ADD] Port project_task_id_display from v8 to v10

### PR DESCRIPTION
**Pull request by ghost**
@victormartinelicocorp @seb-elico @JoJoJoJoJoJoJo  Please review

### CODE REVIEW

**Review by seb-elico — CHANGES_REQUESTED** [review]
Please follow OCA guideline folder structure

File: project_task_id_display/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (elicoidal) [inline project_task_id_display/README.rst]: change license to LGPL and in the manifest

File: project_task_id_display/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
```
Comment (elicoidal) [inline project_task_id_display/__init__.py]: 2015-18

File: project_task_id_display/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (elicoidal) [inline project_task_id_display/__init__.py]: license

File: project_task_id_display/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Task ID Display',
+    'version': '10.0.1.0.0',
+    'category': 'Project',
+    'depends': ['project'],
+    'author': 'Elico Corp',
+    'license': 'AGPL-3',
```
Comment (elicoidal) [inline project_task_id_display/__manifest__.py]: license

File: project_task_id_display/project_task_view.xml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (elicoidal) [inline project_task_id_display/project_task_view.xml]: `<odoo>`

File: project_task_id_display/project_task_view.xml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    <data>
```
Comment (elicoidal) [inline project_task_id_display/project_task_view.xml]: no needed anymore

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details

**Comment by victor-ditay** [comment]
@seb-elico I couldn't found BR for this module migration,  where @elico-shall could input his TMS?

**Comment by ghost** [comment]
Please review @Elico-Corp/odoo-reviewers

**Review by elicoidal — APPROVED** [review]
LGTM

File: project_task_id_display/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2015-18 Elico corp (https://www.elico-corp.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Task ID Display',
+    'version': '10.0.1.0.0',
+    'category': 'Project',
+    'depends': ['project'],
+    'author': 'Elico Corp',
+    'license': 'LGPL-3',
```
Comment (martintorresvm) [inline project_task_id_display/__manifest__.py:10]: please add support key 
```
'support': 'support@elico-corp.com',
```

**Review by victor-ditay — CHANGES_REQUESTED** [review]
LGTM but this

**Comment by ghost** [comment]
@Elico-Corp/odoo-reviewers I have add the 'support': 'support@elico-corp.com' in project_task_id_display/__manifest__.py, please review.

File: project_task_id_display/__init__.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2015-18 Elico corp (https://www.elico-corp.com)
```
Comment (elicoidal) [inline project_task_id_display/__init__.py]: Expect coding line all others are not necessary in `__init__.py` files

File: project_task_id_display/views/project_task_view.xml [hunk 9]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
```
Comment (elicoidal) [inline project_task_id_display/views/project_task_view.xml:6]: add copyright in XML files

**Review by elicoidal — APPROVED** [review]
some small details

**Comment by ghost** [comment]
@Elico-Corp/odoo-reviewers 
1.delete other lines except coding line in __init__.py , 
2.add copyright in XML files, and take records one step forward
3.change 'agpl'  to 'lgpl' in this module

**Review by elicoidal — APPROVED** [review]
Good job!

**Review by victor-ditay — APPROVED** [review]

**Pull request merged**